### ISSUE TITLE
Thread outline for following replies to posts

### ISSUE THREAD

**Issue by Ctrl-R** [sk22/megalodon#256]
![Screenshot_20230103-181706](https://user-images.githubusercontent.com/464787/210463743-6c8d22a5-b3ff-48cb-98b7-c9528f4cb429.png)
As per this screenshot from Fedilab, it would be nice to have a feature that shows the branches of a thread to make it easier to follow.

**Comment by FurbyonSteroid** [comment]
Yup, this is really needed. Not having it makes it really hard to follow conversations. Luckily some apps have it already implemented, but would be a lot cooler if megalodon had it. That's the one major issue I have with the app.
